third_party/blink/renderer/platform/graphics/compositing/paint_artifact_compositor.cc

```diff
@@ -749,6 +750,8 @@ void PaintArtifactCompositor::Update(
     bool backface_hidden = transform.IsBackfaceHidden();
     layer.SetShouldCheckBackfaceVisibility(backface_hidden);
 
+    recordreplay::Assert("[RUN-1470-1471] %d", layer.subtree_property_changed());
```

Comment:
@klochek Please add the name of the function to the `Assert`. That'll allow us to improve on analysis down the line. Because the same `Assert` label might pertain to different functions, we would not be able to easily identify exactly which `Assert`s generate how much overhead etc.